public/app/plugins/datasource/cloudwatch/components/LogsQueryField.test.tsx

```diff
@@ -49,20 +49,19 @@ describe('CloudWatchLogsQueryField', () => {
         onChange={onChange}
       />
     );
-    const getRegionSelect = () => wrapper.find({ label: 'Region' }).props().inputEl;
     const getLogGroupSelect = () => wrapper.find({ label: 'Log Groups' }).props().inputEl;
 
     getLogGroupSelect().props.onChange([{ value: 'log_group_1' }]);
     expect(getLogGroupSelect().props.value.length).toBe(1);
     expect(getLogGroupSelect().props.value[0].value).toBe('log_group_1');
 
     // We select new region where the selected log group does not exist
-    await getRegionSelect().props.onChange({ value: 'region2' });
+    await (wrapper.instance() as CloudWatchLogsQueryField).onRegionChange('region2');
 
     // We clear the select
     expect(getLogGroupSelect().props.value.length).toBe(0);
     // Make sure we correctly updated the upstream state
-    expect(onChange.mock.calls[onChange.mock.calls.length - 1][0]).toEqual({ region: 'region2', logGroupNames: [] });
+    expect(onChange.mock.calls[onChange.mock.calls.length - 1][0]).toEqual({ logGroupNames: [] });
```

Comment:
Is that to test the last call? You could use : `toHaveBeenLastCalledWith` for better readability

```suggestion
    expect(onChange).toHaveBeenLastCalledWith({ logGroupNames: [] });
```

Reply:
Or you can use the calls array to test handler for both line 54 and 59

```suggestion
    expect(onChange.mock.calls).toEqual([[{ logGroupNames: ['log_group_1'] }], [{ logGroupNames: [] }]]);
```